roles/openshift_cloud_provider/meta/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+galaxy_info:
+  author: Sylvain Baubeau
```

Comment:
Not a huge deal currently, but we might need to find a way to attribute authorship to multiple people, and specify who is the maintainer somehow.